components/pages/farms/stake/index.tsx

```diff
@@ -65,30 +60,27 @@ export function Stake({ farm }) {
             }
             // console.log('approved:', current.pid, formatapprove)
 
-
             // LP Staked 
-            let lpStaked = await chadMasterContract.userInfo(current.pid, account).then(amount => {
-                let formattedLPStaked = ethers.utils.formatUnits(amount.amount, 18)
-                var roundedBal = parseFloat(formattedLPStaked).toFixed(2)
-                let abbreviated = numFormatter(roundedBal)
-                console.log('lpstaked ', current.pid, ':', abbreviated)
-
-                if ( abbreviated > 0 && abbreviated < 0.001){
```

Comment:
JS sucks with comparisons. This code works since JS will cast the string to number, but I changed the comparison to use parsed number instead of the "abbreviated" string.

I think there was some bug earlier with strings being compared like this, and it results in situations where:
"3" > "2" == true but also "10" > "2" == false.